src/org/pahappa/systems/models/Type.java

```diff
@@ -0,0 +1,6 @@
+package org.pahappa.systems.models;
+
+public enum Type {
+    REDFLAG,
+    INTERVENTION;
+}
```

Comment:
We already have a Type enum in the enums package. Why did you decide to create another one?

Reply:
Will Delete the File. 
I had created it prior for testing before merging with Benon's work

Reply:
Deleted File, Updated IncidentServiceImpl.java and iReporter.java to reflect the new changes.